SalesforceSDK/Core/Auth/AccountManager.cs

```diff
@@ -67,9 +67,6 @@ public static void DeleteAccount()
         {
             Account account = GetAccount();
             AuthStorageHelper.DeletePersistedCredentials(account.UserName, account.UserId);
-
-            // raise event about account logout
-            AuthenticatedAccountChanged?.Invoke(new AuthenticatedAccountChangedEventArgs(account, null));
```

Comment:
no, when persisted credentials get deleted the event will get raised so we don't need to explicitly call it anymore here.